recipes/bar_chart_race/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "bar_chart_race" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 70c0f6f897a4746472123e509431e75e75a13d5613c27ef8c5b90c8fabdb80b6
+
+build:
+  number: 0
```

Comment:
this can likely be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python